.buildkite/release-pipeline.yaml

```diff
@@ -55,3 +55,18 @@ steps:
           password-env: DOCKERHUB_TOKEN
     env:
       DOCKER_BUILDKIT: "1"
+
```

Comment:
Can we add a block step here, since it only needs to run on-demand (every release)? The syntax is similar to the `Build release image` step.